aloha-core/src/main/scala/com/eharmony/aloha/util/SerializabilityEvidence.scala

```diff
@@ -13,5 +13,5 @@ object SerializabilityEvidence {
 
   implicit def serializableEvidence[A <: java.io.Serializable]: SerializabilityEvidence[A] =
     new SerializabilityEvidence[A]{}
-}
 
+}
```

Comment:
Can you add a newline at the end of the file.

Reply:
I think it has one